expression/builtin_op_vec.go

```diff
@@ -466,11 +470,31 @@ func (b *builtinRealIsFalseSig) vecEvalInt(input *chunk.Chunk, result *chunk.Col
 }
 
 func (b *builtinUnaryMinusIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinUnaryMinusIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+	n := input.NumRows()
+	args := result.Int64s()
+	if mysql.HasUnsignedFlag(b.args[0].GetType().Flag) {
+		for i := 0; i < n; i++ {
+			if uint64(args[i]) > uint64(-math.MinInt64) {
+				return types.ErrOverflow.GenWithStackByArgs("BIGINT", fmt.Sprintf("-%v", uint64(args[i])))
+			}
```

Comment:
We should check if it is equal to `-math.MinInt64` here.

Reply:
@qw4990 I think this check is redundant since the negation of `uint64(-math.MinInt64)` is just `math.MinInt64`. See below:
```
number := int64(math.MinInt64)
fmt.Println(uint64(number) == uint64(-math.MinInt64))
// true
fmt.Println((-number) == math.MinInt64)
// true
```